### PR TITLE
Force secret key on template generation

### PR DESCRIPTION
**Pull request by tbille**
# Summary

For SECRET_KEY on template genreation.
From now on every deplooyement will have an env var SECRET_KEY set.

# QA

- ` ./konf.py production sites/conjure-up.io.yaml --tag test`
- Make sure there is a sercret key in the env var part.

### CODE REVIEW

File: templates/deployment.yaml [hunk 1]
```diff
@@ -24,6 +24,11 @@ spec:
             - name: TALISKER_NETWORKS
               value: 10.0.0.0/8
 
+            - name: SECRET_KEY
+              secretKeyRef:
+                key: {{ data.get('name') or get_site_name() }}
+                name: secret-keys
```
Comment (jpmartinspt) [inline templates/deployment.yaml:30]: If you see above some deployments are using `secret-key`, please just make sure all is now set under `secret-keys`

Comment (tbille) [reply templates/deployment.yaml:30]: yeah I removed all the secret keys from the other services so it uses now the new way of storing the secret keys.

My next task will be to remove manually all the secret keys already in the k8s cluster.

File: sites/360.canonical.com.yaml [hunk 2]
```diff
@@ -17,11 +17,6 @@ env:
       key: database_url
       name: threesixty-database-url
 
-  - name: SECRET_KEY
-    secretKeyRef:
-      key: secret_key
-      name: threesixty
```
Comment (jkfran) [inline sites/360.canonical.com.yaml:23]: @tbille You need to be careful with this one, k8s doesn't allow numbers for deployments but I can see that the secret name you created in our cluster is 360-canonical-com, it should be threesixty-canonical-com.

Comment (tbille) [reply sites/360.canonical.com.yaml:23]: ooooh good catch

**Pull request merged**